pbjson-types/src/timestamp.rs

```diff
@@ -8,14 +8,13 @@ impl TryFrom<Timestamp> for DateTime<Utc> {
     fn try_from(value: Timestamp) -> Result<Self, Self::Error> {
         let Timestamp { seconds, nanos } = value;
 
-        let dt = NaiveDateTime::from_timestamp_opt(
+        Self::from_timestamp(
```

Comment:
This is needed to get clippy passing https://app.circleci.com/pipelines/github/influxdata/pbjson/274/workflows/b620d772-632d-4dd8-bc83-cd9a0e01a30b/jobs/1431

Newer versions of Chrono have deprecated several APIs